File: print-manager-dotnet-core-cs/PageRangeExample.cs

using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    internal class PageRangeExample
    {
        public static void Run()
        {
            FromConstr();
            FromConstr();
        }

        public static void FromConstr()
        {
            PrintJob printJob = new PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"), 1, 3);
            printJob.Print();
        }
        public static void FromAddPage() {
            PrintJob printJob = new PrintJob(Printer.Default);
            printJob.Pages.Add(Util.GetPath("Resources/DocumentA.pdf"), 1, 3);
            printJob.Pages.Add(Util.GetPath("Resources/DocumentA.pdf"), 2, 1);
            printJob.Print();
        }
    }
}

File: print-manager-dotnet-core-vb/PageRangeExample.vb
Imports ceTe.DynamicPDF.Printing


Friend Class PageRangeExample

    Public Shared Sub Run()
        FromConstr()
        FromAddPage()
    End Sub

    Public Shared Sub FromConstr()
        Dim printJob As New PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"), 1, 3)
        printJob.Print()
    End Sub

    Public Shared Sub FromAddPage()
        Dim printJob As New PrintJob(Printer.Default)

        printJob.Pages.Add(Util.GetPath("Resources/DocumentA.pdf"), 1, 3)
        printJob.Pages.Add(Util.GetPath("Resources/DocumentA.pdf"), 2, 1)

        printJob.Print()
    End Sub

End Class
